.github/workflows/build.yml

```diff
@@ -0,0 +1,169 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  bygge_frontend:
+    runs-on: ubuntu-latest
+    outputs:
+      short_sha: ${{ steps.parse_commit.outputs.short_sha }}
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+      - uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: v1-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            v1-yarn-
+      - run: yarn install
+      - run: yarn test
+      - run: yarn build
+      - id: parse_commit
+        run: echo "::set-output name=short_sha::$(git rev-parse --short HEAD)"
```

Comment:
Dette hører ikke helt hjemme her, men skyldes at jeg ikke sjekker ut koden i `lag_versjon`. Kunne eventuelt gjort en `${GITHUB_SHA::8}` på `github.ref` der. Fordelen med `git rev-parse` er at den resolver eventuelle short-sha-kollisjoner.

Reply:
Har samla alt som har med tagging og versjonering i samme jobb, med litt spesifikk logikk for master og !master (PR). Virker ryddigere.